server/pkg/schema/field_select.go

```diff
@@ -0,0 +1,51 @@
+package schema
+
+import "errors"
+
+var TypeSelect Type = "select"
+
+var (
+	ErrFieldValues       = errors.New("invalid values")
+	ErrFieldDefaultValue = errors.New("invalid default values")
+)
+
+type FieldSelect struct {
+	values       *[]string
```

Comment:
```suggestion
	values       []string
```

slice itself is already a reference.